app/code/Bulbulatory/Recommendations/Helper/Email.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Helper;
+
+use Magento\Framework\App\Area;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\App\Helper\AbstractHelper;
+
+/**
+ * Custom Module Email Helper
+ */
+class Email extends AbstractHelper
+{
+    /**
+     * Store manager
+     *
+     * @var StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var StateInterface
+     */
+    protected $inlineTranslation;
+
+    /**
+     * @var TransportBuilder
+     */
+    protected $_transportBuilder;
+
+    /**
+     * @var string
+     */
+    protected $temp_id;
+
+    /**
+     * @var Config
+     */
+    private $configHelper;
+
+    /**
+     * @param Context $context
+     * @param StoreManagerInterface $storeManager
+     * @param StateInterface $inlineTranslation
+     * @param TransportBuilder $transportBuilder
+     * @param Config $configHelper
+     */
+    public function __construct(
+        Context $context,
+        StoreManagerInterface $storeManager,
+        StateInterface $inlineTranslation,
+        TransportBuilder $transportBuilder,
+        Config $configHelper
+    ) {
+        parent::__construct($context);
+
+        $this->_storeManager = $storeManager;
+        $this->inlineTranslation = $inlineTranslation;
+        $this->_transportBuilder = $transportBuilder;
+        $this->configHelper = $configHelper;
+    }
+
+    /**
+     * Return template id according to store
+     *
+     * @param $xmlPath
+     * @return mixed
+     */
+    public function getTemplateId($xmlPath)
+    {
+        return $this->configHelper->getConfigValue($xmlPath);
+    }

```

Comment:
All methods related to fetching module configuration should be put into the `configHelper` so inside `configHelper` should be a method `getTemplateId` which fetches the template and in other classes you can just use `configHelper->getTemplateId()`.

Reply:
@TM18

> All methods related to fetching module configuration should be put into the configHelper so inside configHelper should be a method getTemplateId which fetches the template and in other classes you can just use configHelper->getTemplateId().

Fixed! I created in Config class method **getRecommendationMailTemplateId**